models/cut_semantic_mask_model.py

```diff
@@ -149,7 +149,7 @@ def data_dependent_initialize(self, data):
         self.forward()                     # compute fake images: G(A)
         if self.opt.isTrain:
             self.compute_D_loss().backward()                  # calculate gradients for D
-            self.backward_f_s()
+            self.compute_f_s_loss().backward()                  # calculate gradients for f_s
```

Comment:
If I understand correctly, `self.compute_f_s_loss()` returns nothing. So this may not work? Even if it works for some reason, it's not consistent with other usage  of `self.compute_f_s_loss()`

Reply:
You're right !